ethportal-api/src/types/execution/transaction.rs

```diff
@@ -397,7 +418,7 @@ impl BlobTransaction {
             + self.gas_limit.length()
             + self.to.length()
             + self.value.length()
-            + self.data.len()
+            + self.data.length()
```

Comment:
```suggestion
            + self.data.len()
```

change this back to len() and see if it works

Reply:
It actually doesn't change anything from the perspective of encoding length. I have changed it just to keep it the same for all fields_len methods :)

Reply:
But I tried, and the test still fails.

Reply:
I thought there might be an issue with transaction encoding, but it seems fine. I have added couple of tests if you don't mind

Reply:
Nothing wrong with adding tests

I would dump one that failed with one that didn't fail and output the values as hex. This isn't really my priority to look into what is causing this problem at the moment, but `impl From<Withdrawal> for reth_rpc_types::Withdrawal {` if you remove this and nothing else does it work again?

Reply:
My idea is maybe this is causing a .into() elsewhere in the code to pull this instead for some unknown reason, but that is just my best guess not really looking into this

Reply:
Thank you for your help anyway. I ll look it in coming days